telebot/types.py

```diff
@@ -957,50 +954,50 @@ class Message(JsonDeserializable):
-        received in a message coming through updates, because bot can't be a member of a channel when it is created. It can only 
+    :param channel_chat_created: Optional. Service message: the channel has been created. This field can't be
+        received in a message coming through updates, because bot can't be a member of a channel when it is created. It can only
         be found in reply_to_message if someone replies to a very first message in a channel.
     :type channel_chat_created: :obj:`bool`
 
-    :param message_auto_delete_timer_changed: Optional. Service message: auto-delete timer settings changed in 
+    :param message_auto_delete_timer_changed: Optional. Service message: auto-delete timer settings changed in
         the chat
     :type message_auto_delete_timer_changed: :class:`telebot.types.MessageAutoDeleteTimerChanged`
 
-    :param migrate_to_chat_id: Optional. The group has been migrated to a supergroup with the specified identifier. 
-        This number may have more than 32 significant bits and some programming languages may have difficulty/silent 
-        defects in interpreting it. But it has at most 52 significant bits, so a signed 64-bit integer or double-precision 
+    :param migrate_to_chat_id: Optional. The group has been migrated to a supergroup with the specified identifier.
+        This number may have more than 32 significant bits and some programming languages may have difficulty/silent
+        defects in interpreting it. But it has at most 52 significant bits, so a signed 64-bit integer or double-precision
         float type are safe for storing this identifier.
     :type migrate_to_chat_id: :obj:`int`
 
-    :param migrate_from_chat_id: Optional. The supergroup has been migrated from a group with the specified 
-        identifier. This number may have more than 32 significant bits and some programming languages may have 
-        difficulty/silent defects in interpreting it. But it has at most 52 significant bits, so a signed 64-bit integer or 
+    :param migrate_from_chat_id: Optional. The supergroup has been migrated from a group with the specified
+        identifier. This number may have more than 32 significant bits and some programming languages may have
+        difficulty/silent defects in interpreting it. But it has at most 52 significant bits, so a signed 64-bit integer or
         double-precision float type are safe for storing this identifier.
     :type migrate_from_chat_id: :obj:`int`
 
-    :param pinned_message: Optional. Specified message was pinned. Note that the Message object in this field will not 
+    :param pinned_message: Optional. Specified message was pinned. Note that the Message object in this field will not
         contain further reply_to_message fields even if it is itself a reply.
-    :type pinned_message: :class:`telebot.types.Message`
+    :type message: :class:`telebot.types.Message` or :class:`telebot.types.InaccessibleMessage`
```

Comment:
Supposed to be pinned_message for type
```suggestion
    :type pinned_message: :class:`telebot.types.Message` or :class:`telebot.types.InaccessibleMessage`
```